test-commons/src/main/java/no/unit/nva/publication/PublicationGenerator.java

```diff
@@ -312,5 +406,21 @@ private static Calendar randomDate() {
         calendar.setTime(date);
         return calendar;
     }
+
+    public static  String randomISBN() {
+        return SAMPLE_ISBN;
+    }
+
+
+    public static  PublishingHouse publishingHouseWithUri(URI publisherId) {
+        return new Publisher(publisherId);
+    }
+
+    public static  PublishingHouse unconfirmedPublishingHouse(String publisherName) {
+        return new UnconfirmedPublisher(publisherName);
+    }
```

Comment:
These methods are always called with the methods `randomPublicationChannelsUri()` and `randomString()` respectively.  It seems there is no need for them to have parameters

Reply:
Not yet, are preparing tests to check parameters - will be used in another branch